### DISCUSSION TITLE
Reading session data from Authelia

### DISCUSSION THREAD

**Discussion by lordratner** [authelia/authelia#8295] [Q&A] [answered]
Hi. I'm a simpleton, so I expect this to seem easy to y'all, but I'm stumped. 

I want to use the user information from the Authelia session on the website they logged into with Authelia. How do I do that?

Are the appropriate headers forwarded by default? The site is running on PHP, if that matters. I've tried searching for guides, but I have come up empty. I really just need to pull the username. 

My setup is SWAG for the reverse proxy and Authelia using the authelia/authelia repo. 

Thanks!

**Comment by james-d-elliott** [comment, marked as answer]
This is not something we intentionally provide via the session value, it's opaque and only has meaning to Authelia. The username is provided in the response of the /api/authz/* endpoints or /api/verify endpoint in the `Remote-User` header. You can just use the proxy examples which copy this header from the response to the request for the backend app.
